### PR TITLE
Bryce/add aws manual setup guide

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do?
Add an AWS manual setup guide

### Motivation
DOCS-3234

### Preview
https://docs-staging.datadoghq.com/bryce/add-aws-manual-setup-guide/integrations/guide/aws-integration-manual-setup/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/integrations/guide/aws-integration-manual-setup.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+### Generate an external ID
+Generate an External ID in the [AWS Integration Tile][2] on the integrations page. This will be used on the trust policy of the IAM role you create for Datadog in your AWS account for added security.
+1. Select the `Configuration` tab, then `Role Delegation`.
+2. Click `Manually`. This creates an AWS External ID which is used for configuration of the AWS IAM role. For more information about the External ID, see the [IAM User Guide][3].
+3. Copy this value to your clipboard or notepad.  
+  **Note: Do not close the integration tile or the Datadog application page**, as this causes the external ID value to reset.
+
+### AWS IAM Policy for Datadog
+Create an IAM policy for the Datadog role in your AWS account with [all permissions][4] to take advantage of every AWS integration offered by Datadog. As other components are added to an integration, these permissions may change. If you would like to use Datadog's [Cloud Security Posture Management product][5], you must also add additional permissions by attaching the [AWS SecurityAudit Policy][6] to the role.
```
Comment (warrierr) [inline content/en/integrations/guide/aws-integration-manual-setup.md:43]: Let's move the Cloud Security Posture Management call our to the IAM role creation section, and have an explicit optional step to add the SecurityAudit policy to the role as well

File: content/en/integrations/guide/aws-integration-manual-setup.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+10. Create a new role in the AWS [IAM Console][8].
+11. Select `Another AWS account` for the Role Type.
+12. For Account ID, enter `464622532012` (Datadog's account ID). This means that you are granting Datadog access to your AWS data.
+13. Select `Require external ID` and enter the one generated in [AWS External ID](#generate-an-external-id). 
+Make sure you leave **Require MFA** disabled. 
+14. Click `Next`.
+15. If you've already created the policy, search for it on this page and select it. Otherwise, click `Create Policy`, which opens in a new window, and follow the instructions from [AWS IAM Policy for Datadog](#aws-iam-policy-for-datadog). 
+16. Click `Next: Tags` and `Next: Review`.
+17. Give the role a name such as `DatadogAWSIntegrationRole`, as well as an apt description.
```
Comment (warrierr) [inline content/en/integrations/guide/aws-integration-manual-setup.md:63]:
```suggestion
17. Give the role a name such as `DatadogIntegrationRole`, as well as an apt description. 
```

File: content/en/integrations/guide/aws-integration-manual-setup.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+16. Click `Next: Tags` and `Next: Review`.
+17. Give the role a name such as `DatadogAWSIntegrationRole`, as well as an apt description. 
+18. Click `Create Role`.
+
+### Complete the setup in Datadog
+
+19. Returning to the [AWS integration tile][2] page in your Datadog account that you had open in another tab, enter your AWS Account ID **without dashes**, for example: `123456789012`. Your Account ID can be found in the ARN of the role created for Datadog.
+20. Enter the name of the created role. **Note:** The role name you enter in the integration tile is case sensitive and must exactly match the role name created on the AWS side.
+21. If there is a [Datadog is not authorized to perform sts:AssumeRole][9] error, make sure your AWS trust policy's `sts:ExternalId:` matches the [`AWS External ID`](#generate-an-external-id) previously created in the integration tile.
+22. Optionally, customize the AWS services to collect metrics from on the left side of the dialog.
```
Comment (warrierr) [inline content/en/integrations/guide/aws-integration-manual-setup.md:71]: Let's remove all these "Optionally" steps. Re-reading this, I think it is a bit confusing without explaining more about what/where each of these are, so I think we should just remove

File: content/en/integrations/guide/aws-integration-manual-setup.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+23. Optionally, add tags to all hosts and metrics.
+24. Optionally, monitor a subset of EC2 instances by entering the AWS tags in the textbox `to hosts with tag`. **Note:** This also applies to an instance's attached EBS volumes.
+25. Optionally, monitor a subset of Lambdas by entering the AWS tags in the textbox `to Lambdas with tag`.
+26. Click **Install Integration**.
+
```
Comment (warrierr) [inline content/en/integrations/guide/aws-integration-manual-setup.md:76]: Let's add a step like we have in the getting started guide: 
`Wait up to 10 minutes for data to start being collected, and then view the out-of-the-box [AWS overview dashboard](https://app.datadoghq.com/screen/integration/7/aws-overview) to see metrics sent by your AWS services and infrastructure`

File: content/en/integrations/guide/aws-integration-manual-setup.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: /integrations/amazon_web_services/
```
Comment (warrierr) [inline content/en/integrations/guide/aws-integration-manual-setup.md:79]: Can we add links to the docs/guides for lambda forwarder, kinesis firehose for logs, and metric streams to these links as well?

**Review by warrierr — CHANGES_REQUESTED** [review]

**Review by warrierr — CHANGES_REQUESTED** [review]
Also: Update all links to open in new tabs

Link to new permissions section on main page once available

**Comment by buraizu** [comment]
Closing in favor of PR 13963

**Pull request closed without merging**